lib/compress/zstd_compress.c

```diff
@@ -385,6 +385,11 @@ ZSTD_bounds ZSTD_cParam_getBounds(ZSTD_cParameter param)
         bounds.upperBound = ZSTD_lcm_uncompressed;
         return bounds;
 
+    case ZSTD_c_targetCBlockSize:
+        bounds.lowerBound = 0; // TODO probably need lower bound
```

Comment:
Good point !
Yes, very likely, and it will have to be documented.

Not obvious to decide one.
For a limit, we don't need to force a certain level of efficiency.
We merely need a limit which avoids triggering weird side effects in the implementation, and does not constrain the user too much. So it's likely going to be small.

Probably something smaller than 100 bytes, like 50, or 64 bytes.
I have no specific preference. I just seems the right ballpark.